base/toml_parser.jl

```diff
@@ -925,14 +925,16 @@ ok_end_value(c::Char) = iswhitespace(c) || c == '#' || c == EOF_CHAR || c == ']'
 accept_two(l, f::F) where {F} = accept_n(l, 2, f) || return(ParserError(ErrParsingDateTime))
 function parse_datetime(l)
     # Year has already been eaten when we reach here
-    year = parse_int(l, false)::Int64
+    year = parse_int(l, false)
```

Comment:
```suggestion
    year = @try parse_int(l, false)
```

and remove the check below I think. Can do this in the other places as well.

Reply:
done